controllers/ProfileController.js

```diff
@@ -36,8 +36,51 @@ const getUserLikes = async (req, res) => {
 
 const getProfileContents = async (req, res) => {
   try {
-    const { username } = req.query;
+    const { userId, username } = req.query;
```

Comment:
This is the profile owner's id right (ie. user A is looking at user B's profile, and this is user B's user id)? Can we please rename this variable to make it clearer?

Reply:
I have updated it so that it is like such: 

```suggestion
    const { currentUserId, visitedUsername } = req.query;
```